azure/README.md

```diff
@@ -4,6 +4,8 @@
 confluent-hub install confluent/kafka-config-provider-azure:latest
 ```
 
+Note: if this command does not work and the package is not online available to download, then you need to download the zip file for azure and include it manually in the plugins path, for example for azure 1.0.5 you would get confluent-csid-secrets-provider-azure-1.0.5.zip from https://github.com/confluentinc/csid-secrets-providers/releases/tag/csid-secrets-providers-1.0.5, but look whatever version is best for you
```

Comment:
the last section after the comma reads a little strange?

Reply:
deleted that part